packages/stacking/src/index.ts

```diff
@@ -0,0 +1,410 @@
+// import axios from 'axios';
+// import { BigNumber } from 'bignumber.js';
+import {
+  makeContractCall,
+  bufferCV,
+  uintCV,
+  tupleCV,
+  ClarityType,
+  broadcastTransaction,
+  standardPrincipalCV,
+  ContractCallOptions,
+  UIntCV,
+  BufferCV,
+  ContractCallPayload,
+  StacksTransaction,
+  callReadOnlyFunction,
+  cvToString,
+  ClarityValue,
+  ResponseErrorCV,
+  SomeCV,
+  TupleCV
+} from '@stacks/transactions';
+import {
+  StacksNetwork,
+  StacksMainnet
+} from '@stacks/network';
+import BN from 'bn.js';
+import { StackingErrors } from './constants';
+import { fetchPrivate } from '@stacks/common';
+import { convertBTCAddress } from './utils';
+
+export interface PoxInfo {
+  contract_id: string;
+  first_burnchain_block_height: number;
+  min_amount_ustx: string;
+  registration_window_length: 250;
+  rejection_fraction: number;
+  reward_cycle_id: number;
+  reward_cycle_length: number;
+  rejection_votes_left_required: number;
+}
+
+export interface StackerInfo {
+  amountMicroStx: string;
+  firstRewardCycle: number;
+  lockPeriod: number;
```

Comment:
I'm assuming this method will be called to show either # of blocks until unlocked or timestamp of unlock event. Could we calculate these and return them as well for convenience? Something like:

- secondsUntilUnlock
- unlockBlockHeight

Reply:
That would take an extra network call to fetch block info in order to calculate. Probably best to let the developer do that separately if they need it.

Reply:
well, I guess the question is how would they use this method. My assumption is it'll be used to show the data. It's probably fair to say the progress or unlocking block/time would be important to show. 

could we allow an optional param to request this data, eg `getStatus(unlockDetails: boolean)`?

Reply:
It would be better as a separate method then

Reply:
could be if devs would want to call that outside of the getstatus flow. either, way, I think that info should be easily available to them

Reply:
the `getStatus` function is now making an additional call to the account API and returns the unlock block as part of the response